modules/core/keeper/keeper_test.go

```diff
@@ -0,0 +1,124 @@
+package keeper_test
+
+import (
+	"testing"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	capabilitykeeper "github.com/cosmos/cosmos-sdk/x/capability/keeper"
+	"github.com/stretchr/testify/suite"
+
+	stakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	upgradekeeper "github.com/cosmos/cosmos-sdk/x/upgrade/keeper"
+	clienttypes "github.com/cosmos/ibc-go/v3/modules/core/02-client/types"
+	ibchost "github.com/cosmos/ibc-go/v3/modules/core/24-host"
+	ibckeeper "github.com/cosmos/ibc-go/v3/modules/core/keeper"
+	ibctesting "github.com/cosmos/ibc-go/v3/testing"
+	"github.com/cosmos/ibc-go/v3/testing/simapp"
+)
+
+type KeeperTestSuite struct {
+	suite.Suite
+
+	app           *simapp.SimApp
+	stakingKeeper clienttypes.StakingKeeper
+	upgradeKeeper clienttypes.UpgradeKeeper
+	scopedKeeper  capabilitykeeper.ScopedKeeper
+}
```

Comment:
I think I'd prefer to use the existing `KeeperTestSuite` (as defined in msg_server_test). We can move the declaration existing `KeeperTestSuite` to this file